projects/demo/src/modules/components/editor/images/upload/examples/1/index.html

```diff
@@ -1,11 +1,20 @@
-<tui-loader [showLoader]="!!(imgbbService.loading$ | async)">
+<tui-loader
+    [overlay]="true"
+    [showLoader]="!!(imgbbService.loading$ | async)"
+>
     <tui-editor
         class="editor"
         [formControl]="control"
         [tools]="builtInTools"
+        (blur)="control.updateValueAndValidity()"
```

Comment:
Why do we need to trigger it manually? Should `onTouched` from `ControlValueAccessor` take care of that? Also you seem to already be doing that inside component template by newly added focus/blur handlers.

Reply:
good catch, I forgot remove that line

Reply:
fixed